crowd_anki/anki_exporter.py

```diff
@@ -42,8 +42,7 @@ def export_deck_to_directory(self, deck_name, output_dir=Path("."), copy_media=T
                                        default=Deck.default_json,
                                        sort_keys=True,
                                        indent=4,
-                                       ensure_ascii=False,
```

Comment:
does this still work properly on 2.0 if your system default encoding is not utf8?

Reply:
I think so, because the default parameter is set to UTF-8. 
Source: https://docs.python.org/2/library/json.html